template/istqb.cls

```diff
@@ -53,6 +53,117 @@
 \usepackage{parskip}  % Put half of line height between paragraphs.
 \sloppy  % Prevent overfull lines.
 
+%% Language
+\PassOptionsToPackage{shorthands=off}{babel}
+\RequirePackage[czech,slovak,english]{babel}
+\ExplSyntaxOn
+\tl_new:N \g_istqb_language_tl
+\prop_new:N \g_istqb_reference_labels_prop
+\cs_new:Nn
+  \istqb_set_language:n
+  {
+    \tl_gset:Nn
+      \g_istqb_language_tl
+      { #1 }
+    \prop_gclear:N
+      \g_istqb_reference_labels_prop
+    \str_case:nnF
+      { #1 }
+      {
+        { cs } {
+          \selectlanguage { czech }
+          \gdef \istqbstandardsname { Standardy }
+          \gdef \istqbdocumentsname { ISTQB\textsuperscript{\textregistered}~dokumenty }
+          \gdef \istqbbooksname { Knihy }
+          \gdef \istqbarticlesname { Články }
+          \gdef \istqbfurtherreadingname { Další~zdroje }
```

Comment:
I see that `References` is translated as `Odkazy` in czech and `Referencie` in Slovak, despite not seeing it here. Can we override default translation to `Reference` and `Referencie`?

Reply:
> despite not seeing it here

Some of the translations are provided by the LaTeX babel package.

> Can we override default translation to `Reference` and `Referencie`?

Definitely.

There may be value in extracting the translations into separate YAML files, so that new languages can be added easily by non-programmers. It would take an extra ~1 hour of effort on my part now but would save effort in future translations.

Reply:
> There may be value in extracting the translations into separate YAML files, so new languages can be added easily by non-programmers. It would take an extra ~1 hour of effort on my part now but would save effort in future translations.

@danopolan Do you think this would be valuable?

Reply:
I added the YML files in commit 64b0948 from this PR.